Issues 7

michel-heon opened the pull request:
Associé à [https://github.com/dsi-ehess/VIVO-EHESS/issues/7](url)

Commit:
Remplacer "ontology" par "vivo" dans le nom de fichier pour désigner
l'ontologie vivo de l'EHESS

Commit:
Ajout des IRI ORG_00000{16,21} à vivo_fr_FR et adaptation du menu
déroulant pour orgType dans 'créer une fonction'

Commit:
Ajout de CNRS aux organisations apparaissant dans le menu

j-dornbusch reviewed (changes requested):
@michel-heon Il y a un malentendu : les quelques cas donnés sont des exemples (des instances). Il faudrait trouver quelques classes générales (type "établissement public" ou "association" ou "entreprise") dans les ontologies utilisées par Vivo ou ajouter des classes ad hoc dans l'ontologie française, mais ce dernier choix serait étrange, puisque rien de tout cela n'est spécifique à la France. 
En tout cas on ne peut pas à mon sens faire figurer ces quelques exemples, pris au hasard parmi des milliers de possibles, dans l'ontologie.

Commit:
Création de la classe EHESS-Affiliation de type foaf:Organization et
gestion de ses sous-classes afin qu'elles apparaissent dans le menu
déroulant 'type d'organisation' de la page ajouter un fonction.

Commit:
Correction des tabs par des whitespaces

MichelHeon commented:
@j-dornbusch 
Avec cette correction, toutes les organisations sous 'EHESS-Affiliation' seront dans le menu déroulant.
J'ai ajouté une ontologie de test (/home/src/main/resources/rdf/abox/filegraph/vivo_fr_FR_EHESS_AFFILIATION.n3)  qui sera à supprimer avant le merge.
